test/unit/controllers/ApplicationControllerSpec.scala

```diff
@@ -95,13 +93,38 @@ class ApplicationControllerSpec extends ControllerSpec with BeforeAndAfterEach {
       }
     }
 
+    "direct authenticated users where group id has CDS enrolment to start short-cut subscription" in {
+      when(enrolmentStoreProxyService.isEnrolmentAssociatedToGroup(any(), ArgumentMatchers.eq(ATaR))(any()))
+        .thenReturn(Future.successful(false))
+      when(enrolmentStoreProxyService.isEnrolmentAssociatedToGroup(any(), ArgumentMatchers.eq(CDS))(any()))
+        .thenReturn(Future.successful(true))
+
+      val result =
```

Comment:
Application with invoking method didn't handle exceptions correctly. I needed to extract this part from the method and stop using it.